libcaf_net/test/net/basp/message_queue.cpp

```diff
@@ -0,0 +1,155 @@
+/******************************************************************************
+ *                       ____    _    _____                                   *
+ *                      / ___|  / \  |  ___|    C++                           *
+ *                     | |     / _ \ | |_       Actor                         *
+ *                     | |___ / ___ \|  _|      Framework                     *
+ *                      \____/_/   \_|_|                                      *
+ *                                                                            *
+ * Copyright 2011-2019 Dominik Charousset                                     *
+ *                                                                            *
+ * Distributed under the terms and conditions of the BSD 3-Clause License or  *
+ * (at your option) under the terms and conditions of the Boost Software      *
+ * License 1.0. See accompanying files LICENSE and LICENSE_ALTERNATIVE.       *
+ *                                                                            *
+ * If you did not receive a copy of the license files, see                    *
+ * http://opensource.org/licenses/BSD-3-Clause and                            *
+ * http://www.boost.org/LICENSE_1_0.txt.                                      *
+ ******************************************************************************/
+
+#define CAF_SUITE basp.message_queue
+
+#include "caf/net/basp/message_queue.hpp"
+
+#include "caf/test/dsl.hpp"
+
+#include "caf/actor_cast.hpp"
+#include "caf/actor_system.hpp"
+#include "caf/behavior.hpp"
+
+using namespace caf;
+
+namespace {
+
+behavior testee_impl() {
+  return {[](ok_atom, int) {
+    // nop
+  }};
```

Comment:
```suggestion
  },};
```

Aside from other reasons, `clang-format` will format message handlers better if you're using a trailing comma.